src/schemas/CustomerChargeFiltersUsageObject.yaml

```
@@ -13,6 +14,11 @@ items:
       pattern: "^[0-9]+.?[0-9]*$"
       example: "0.9"
       description: The number of units consumed for a specific charge filter related to a charge item.
+    total_aggregated_units:
+      type: string
+      pattern: "^[0-9]+.?[0-9]*$"
```

I think this would be more correct as 
```suggestion
      pattern: "^[0-9]+(\.[0-9]+)?$"
```

---

Should we change `units` as well? It is the same data type.